src/test/v1.2/advanced/fdc3.open.ts

```diff
@@ -26,15 +25,15 @@ export default () =>
       const result = control.contextReceiver("fdc3-conformance-opened");
       await control.openMockApp(openApp.b.name);
       await result;
-      await closeMockAppWindow(AOpensB2Test);
+      await control.closeMockApp(AOpensB2Test);
```

Comment:
remove the control.closeMockApp and just use closeMockAppWindow?

Reply:
I remember now why I'm calling the closeMockApp function via the interface implementation instead of directly from the utils folder. It's because the common tests require the function to be called from the 1.2 utils file or the 2.0 utils file depending on whether 1.2 or 2.0 tests are being run.

using the interface, the tests need to be able to freely switch between the two versions of the function

Reply:
Well worth adding a comment then (Future-Joe will thank you!)